src/pygments_djc/__init__.py

```diff
@@ -0,0 +1,22 @@
+from pygments.lexers import LEXERS
+
+# Public API
+from pygments_djc.lexers import DjangoComponentsPythonLexer
+
+__all__ = ["DjangoComponentsPythonLexer"]
+
+
+# Register the Lexer. Unfortunately Pygments doesn't support registering a Lexer
+# without it living in the pygments codebase.
+# See https://github.com/pygments/pygments/issues/1096#issuecomment-1821807464
+LEXERS["DjangoComponentsPythonLexer"] = (
```

Comment:
As mentioned here - Pygments has an (IMO) strange design, where they keep all the Lexers as part of the codebase and part of the package, and don't provide some official way to add own Lexers. So we have to do so here.

Thus, when one imports `pygments_djc`, our Lexer will be added.